adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -407,10 +413,12 @@ public Guid CorrelationId
             return await handler.RunAsync().ConfigureAwait(false);
         }
 
-        internal IWebUI CreateWebAuthenticationDialog(IPlatformParameters parameters)
+#if !NETSTANDARD1_3
```

Comment:
>NETSTANDARD1_3 [](start = 5, length = 14)

are these ok for NETSTANDARD1_1 ?

Reply:
1.1 and 1.3 don't show UI, so actually need to add 1.1 here and remove PlatformParameters and WebUIFactory from those respective platforms in ADAL. nice catch.

Reply:
xamarin is not picking up on the platforms w/the ifdefs, so going to revert this back to original, with throwing a `NotImplementedException()` - and removing the idefs from `AuthenticationContext`